cran-comments.md

```diff
@@ -1,27 +1,46 @@
 # Description
 
-This submission features a bug patch and a few minor documentation changes.
-
 Thank you for taking the time to review this submission and please reach out to either Stephan Kadauke or Richard Hanna for any questions, comments, or concerns.
 
-## R CMD check results:
+This submission includes new features, optimizations, and patches for a few identified bugs.
 
-- 0 ERRORs or WARNINGs on any builds
-- A NOTE is generated in R-hub Windows (Server 2022, R-devel 64-bit), a similar issue appears to arise in the `REDCapR` package and appears to be related to the R-hub test environment.
+### New Features
 
-```
-* checking for detritus in the temp directory ... NOTE
-Found the following files/directories:
-  'lastMiKTeXException'
-```
+* Function name changes:
+  * `read_redcap_tidy` is now `read_redcap`
+  * `extract_table` is now `extract_tibble`
+  * `extract_tables` is now `extract_tibbles`
+  * `bind_table` is now `bind_tibbles`
+    * All old functions are still supported and deliver a deprecation warning
+* The `read_redcap` output now includes additional helpful metadata
+* New `make_labelled()` function allowing for descriptive labelling application to variables via the [`labelled`](https://larmarange.github.io/labelled/) package
+* New arguments available for `read_redcap` capabilities:
+  * `forms`
+  * `export_survey_fields`
+
+## Test environments
 
+1. Local Ubuntu, R 4.2.0
```

Comment:
Why local Ubuntu? Should this say Local macOS?

Reply:
Correct, my bad